### PR TITLE
app-doc/eclass-manpages: display @SUPPORTED_EAPIS

### PR DESCRIPTION
**Pull request by juippis**

### CODE REVIEW

File: app-doc/eclass-manpages/files/eclass-to-manpage.awk [hunk 1]
```diff
@@ -90,10 +90,9 @@ function xfail(text) {
 	_stderr_msg(text, "error (ignoring)")
 	exit(77)
 }
-
 function eat_line() {
 	ret = $0
-	sub(/^# @[A-Z]*:[[:space:]]*/,"",ret)
+	sub(/^# @.*:[[:space:]]*/,"",ret)
```
Comment (juippis) [inline app-doc/eclass-manpages/files/eclass-to-manpage.awk]: Note: I originally created a new function called `eat_eapis`, without touching this eat_line sub, but decided to give this a try and it didn't seem to break anything with the 10 eclasses I tried it with. No knowledge about **possible** breakages, that's why it's up for review ;)

```
function eat_eapis() {
	eapiret = $0
	sub(/^# @SUPPORTED_EAPIS*:[[:space:]]*/,"",eapiret)
	getline
	return eapiret
}
```

**Comment by juippis** [comment]
https://gist.github.com/juippis/95f801437239964b8639924b2048c43e

File: app-doc/eclass-manpages/files/eclass-to-manpage.awk [hunk 2]
```diff
@@ -90,10 +90,9 @@ function xfail(text) {
 	_stderr_msg(text, "error (ignoring)")
 	exit(77)
 }
-
```
Comment (mgorny) [inline app-doc/eclass-manpages/files/eclass-to-manpage.awk]: Why did you eat this line?

Comment (juippis) [reply app-doc/eclass-manpages/files/eclass-to-manpage.awk]: Oops, that's where I had the original `eat_eapis` function. Functions above (short ones) aren't separated by a newline, but functions below (long ones) are. I must've looked example from above :P 

thanks!

Comment (mgorny) [reply app-doc/eclass-manpages/files/eclass-to-manpage.awk]: I think they're actually grouped, and groups are separated.

**Pull request closed without merging**